### DISCUSSION TITLE
EISDIR: illegal operation on a directory, read

### DISCUSSION THREAD

**Discussion by yu-steven** [lowlighter/metrics#869] [Q&A]
Hi everyone ！I have a problem about this action !
I have a openit Storehouse，but this warehouse has nothing to do with this action, so why does it report an error?
https://github.com/git-yusteven/git-yusteven/runs/5150400127?check_suite_focus=true

**Comment by lowlighter** [comment]
Hi 👋 !

Seems to be an issue with the [`🈷️ language`](https://github.com/lowlighter/metrics/tree/master/source/plugins/languages) plugin. 

The workflow run has been deleted since, but I assume you were using `plugin_languages_indepth` which as mentioned in the [documentation](https://github.com/lowlighter/metrics/tree/master/source/plugins/languages#-indepth-mode) clones your repository on the worker to analyze them thoroughly with [linguist-js](https://github.com/Nixinova/linguist). 

But it looks like that somewhere in the code it attempted to `fs.readFile` a directory but not sure why unfortunately 😕 

If you have a trouble with a single repository, you could add it to `plugin_languages_skipped` to ignore it as a workaround.

To fix it it would be nice to have a repro though 🤔

**Reply by yu-steven** [reply]
Yea thank you for reply,I deleted the previous link.
I will give you a picture and openit is a project.
Is it ok?
![9F02331B-6B92-4986-AF58-E2E6A6214FD8](https://user-images.githubusercontent.com/78414278/153691845-cb8eb01b-1153-42a5-970b-f765e356c7cd.jpeg)

**Reply by lowlighter** [reply]
Ok thanks, I'll try to reproduce it later 👍

**Reply by yu-steven** [reply]
Nice Thank you

**Reply by lowlighter** [reply]
Just a question, did it happen systematically, or just sometimes ?

**Reply by yu-steven** [reply]
Systematic, because I set it to update once every hour, I reminded me of an action error every hour since I created a new file called url and deleted it.

**Reply by yu-steven** [reply]
But it doesn't matter, because I have found an alternative!

I just want to know the reason for the error so that the next setup will be smoother.
